app/src/main/java/com/github/h3lp3rs/h3lp/forum/implementation/FireDBForum.kt

```diff
@@ -11,25 +10,4 @@ import com.github.h3lp3rs.h3lp.forum.Path
  * @param path The path of the current forum pointer.
  * @database The FireDatabase that serves as root.
  */
-class FireDBForum (
-    override val path: Path,
-    private val database: FireDatabase
-) :
-    SimpleDBForum(database) {
-
-    override fun root(): Forum {
-        return FireDBForum(emptyList(), database)
-    }
-
-    override fun child(relativePath: Path): Forum {
-        return FireDBForum(path + relativePath, database)
-    }
-
-    override fun parent(): Forum {
-        return if (isRoot()) {
-            this
-        } else {
-            FireDBForum(path.dropLast(1), database)
-        }
-    }
-}
\ No newline at end of file
+open class FireDBForum(path: Path, database: FireDatabase) : SimpleDBForum(path, database)
```

Comment:
I'm pretty sure you don't need the open keyword here (since the class won't be inherited from)

Reply:
Ow nice catch, this is simply a mistake (I opened the wrong class at first and forgot changing it)